### ISSUE TITLE
Events from subscribed calendar not shown

### ISSUE THREAD

**Issue by manfredu** [nextcloud/calendar#1899]
### Steps to reproduce
1. Add new subscription via url https://www.tiefbauamt.bs.ch/dam/jcr:4b2be79d-8325-411f-81f8-5ee339cee7ab/Zone%20H_2020.ics 
2. Wait until cron job has run or run manually 
3. Refresh display of calender

### Expected behaviour
Events should be displayed

### Actual behaviour
No entry at all from this calendar link.
But there are objects in table calenderobjects. E.g. one row contains 
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//Sabre//Sabre VObject 4.1.6//EN
CALSCALE:GREGORIAN
BEGIN:VEVENT
DTSTAMP:20191009T120827Z
DTSTART;VALUE=DATE:20200115
SUMMARY:Grobsperrgut
UID:91f27aa8-fa64-4981-b52d-51ba3f191d4c
X-MICROSOFT-CDO-ALLDAYEVENT:TRUE
LOCATION:Zone H
DESCRIPTION:Grobsperrgut Zone H
CATEGORIES:Zone H Grobsperrgut
END:VEVENT
END:VCALENDAR

Shouldn't that be displayed on the 15.01.? It is not....

### Calendar app
2.0.1

**CalDAV-clients used:** DAVx5

### Client configuration
Vivaldi | 2.10.1745.27 (Stable channel) (64-Bit)
Firefox 72.0.1

**Operating system:** Ubuntu 19.10

### Server configuration

**Operating system**: FreeNAS 11.3 RC2

**Web server:** Caddy/v1.0.4

**Database:** 10.3.20-MariaDB

**PHP version:** 7.3.13

**Nextcloud Version:** 17.0.2

**Updated from an older installed version or fresh install:** Updated from latest before 2.0.0

**List of activated apps:**
```
Enabled:
  - accessibility: 1.3.0
  - activity: 2.10.1
  - announcementcenter: 3.6.1
  - apporder: 0.9.0
  - calendar: 2.0.1
  - circles: 0.17.10
  - cloud_federation_api: 1.0.0
  - cms_pico: 1.0.4
  - comments: 1.7.0
  - contacts: 3.1.8
  - dav: 1.13.0
  - deck: 0.8.0
  - drawio: 0.9.4
  - encryption: 2.5.0
  - external: 3.4.1
  - federatedfilesharing: 1.7.0
  - federation: 1.7.0
  - files: 1.12.0
  - files_pdfviewer: 1.6.0
  - files_rightclick: 0.15.1
  - files_sharing: 1.9.0
  - files_trashbin: 1.7.0
  - files_versions: 1.10.0
  - files_videoplayer: 1.6.0
  - firstrunwizard: 2.6.0
  - gallery: 18.4.0
  - groupfolders: 5.0.5
  - logreader: 2.2.0
  - lookup_server_connector: 1.5.0
  - metadata: 0.11.1
  - nextcloud_announcements: 1.6.0
  - notes: 3.1.1
  - notifications: 2.5.0
  - oauth2: 1.5.0
  - password_policy: 1.7.0
  - privacy: 1.1.0
  - provisioning_api: 1.7.0
  - recommendations: 0.5.0
  - serverinfo: 1.7.0
  - sharebymail: 1.7.0
  - spreed: 7.0.2
  - support: 1.0.1
  - survey_client: 1.5.0
  - systemtags: 1.7.0
  - tasks: 0.11.3
  - text: 1.1.1
  - theming: 1.8.0
  - twofactor_backupcodes: 1.6.0
  - updatenotification: 1.7.0
  - viewer: 1.2.0
  - workflowengine: 1.7.0
Disabled:
  - admin_audit
  - files_external
  - mail
  - user_ldap
```
**Nextcloud configuration:**
```
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost",
            "cloud-basel.usselmann.it",
            "192.168.178.100"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "17.0.2.1",
        "overwrite.cli.url": "https:\/\/cloud-basel.usselmann.it\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "Europe\/Berlin",
        "log_type": "file",
        "logfile": "\/var\/log\/nextcloud.log",
        "loglevel": "2",
        "logrotate_size": "104847600",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0
        },
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "htaccess.RewriteBase": "\/",
        "maintenance": false,
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "tls",
        "mail_smtpauth": 1,
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "PLAIN"
    }
}
```
### Logs
[nextcloud_errors.log](https://github.com/nextcloud/calendar/files/4109716/nextcloud_errors.log)
[nextcloud.log](https://github.com/nextcloud/calendar/files/4109718/nextcloud.log)

![image](https://user-images.githubusercontent.com/50785/73089359-564c8400-3ed6-11ea-8487-503ba78ef878.png)
![image](https://user-images.githubusercontent.com/50785/73089392-6c5a4480-3ed6-11ea-8fef-9bd1d218c0bb.png)

**Comment by albatros69** [comment]
Seems to be same issue I'm facing in #1896

**Comment by georgehrke** [comment]
@manfredu Thx for providing that link, i was able to reproduce the error.

Working on a fix ...

**Comment by georgehrke** [comment]
Fix is in: https://github.com/georgehrke/calendar-js/pull/77

**Comment by georgehrke** [comment]
@manfredu @albatros69 I checked both the calendar links you provided and they work after I applied the fix mentioned above.

I will go ahead and close this issue. The fix will be released with version 2.0.2

**Comment by manfredu** [comment]
> @manfredu @albatros69 I checked both the calendar links you provided and they work after I applied the fix mentioned above.
> 
> I will go ahead and close this issue. The fix will be released with version 2.0.2

Thank you! :-)

**Comment by albatros69** [comment]
> I will go ahead and close this issue. The fix will be released with version 2.0.2

Thanks for the support and the fix!
